zio-http/shared/src/main/scala/zio/http/Response.scala

```diff
@@ -58,6 +58,9 @@ final case class Response(
         self.copy(body = Body.fromChunk(bytes))
       }
 
+  def contentType(mediaType: MediaType): Response =
+    self.addHeader("content-type", mediaType.fullType)
```

Comment:
Outside the scope of this PR, but I'm wondering whether we should change the `fullType` to a `val` instead of a `lazy val` to avoid the penalty of lazy vals.

Reply:
Since both are static strings, I'll expect the val to be cheap anyway

Reply:
If this would be scala 3 only, inline def is probably best?

Reply:
Let's tackle this in another issue/PR where we can revise the usages of `lazy val` in ZIO HTTP